src/assets/i18n/en.json5

```diff
@@ -4811,4 +4811,7 @@
   "person.orcid.registry.auth": "ORCID Authorizations",
   "home.recent-submissions.head": "Recent Submissions",
 
+  "idle-modal.extend-session": "Extend session",
```

Comment:
This appears to be a duplicative i18n key.  It already exists on line 4582, so it can be removed from this line